### PR TITLE
Support pipes and services

### PR DESCRIPTION
**Pull request by ftr93**

### CODE REVIEW

**Review by Etienne-Buschong — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "storybook-webpack-angular-types-plugin",
-	"version": "0.4.3",
+	"version": "0.5.0",
```
Comment (kroeder) [inline package.json:3]: @ftr93 you pushed a tag commit I think

Can you revert this, please?

File: packages/angular-demo/src/app/example.service.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+		return this._val;
+	}
+
+	ngOnInit() {}
+
+	ngOnChanges() {}
+
+	ngAfterViewInit() {}
+
+	ngAfterContentInit() {}
```
Comment (kroeder) [inline packages/angular-demo/src/app/example.service.ts:109]: Angular services only support `ngOnDestroy`

File: packages/angular-demo/src/app/example.service.ts [hunk 3]
```diff
@@ -0,0 +1,126 @@
+/* eslint-disable */
+// noinspection JSUnusedGlobalSymbols,JSUnusedLocalSymbols,JSMethodCanBeStatic
+
+import { Injectable } from '@angular/core';
+import { NestedInterface, TestInterface, TestType } from './types';
+
+@Injectable()
+export class ExampleService {
```
Comment (kroeder) [inline packages/angular-demo/src/app/example.service.ts:8]: For the sake of completeness, you should probably add `implements OnDestroy`

**Review by kroeder — CHANGES_REQUESTED** [review]
Can you remove the tag commit before I merge the PR, please?

**Review by kroeder — APPROVED** [review]

**Pull request merged**